httpfuzzerprocessor/showDifferences.js

```diff
@@ -0,0 +1,76 @@
+//A Fuzzer HTTP Processor script that compares the original Response with the fuzzed Response 
+//and add the result to the state column!
+ 
+var DiffTool = Java.type("org.zaproxy.zap.extension.diff.diff_match_patch");
+var key = "fuzz.httpfuzzer.messageprocessor.tagcreator.tags.text";
+var showResultInTable = true;
```

Comment:
The intention of this named boolean variable is to avoid the comment
` // Whether the result should be accepted, or discarded and not shown.`

But I can remove this variable and return directly `true`...